pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -36,8 +38,35 @@ func NewFlowRecords(connStore *connections.ConnectionStore) *FlowRecords {
 
 // BuildFlowRecords builds the flow record map from connection map in connection store
 func (fr *FlowRecords) BuildFlowRecords() error {
+	addOrUpdateFlowRecord := func(key flowexporter.ConnectionKey, conn flowexporter.Connection) error {
+		// If DoExport flag is not set return immediately.
+		if !conn.DoExport {
+			return nil
+		}
+		record, exists := fr.recordsMap[key]
+
+		isIPv6 := false
+		if net.ParseIP(key[0]).To4() == nil {
+			isIPv6 = true
+		}
+		if !exists {
+			record = flowexporter.FlowRecord{
+				Conn:               &conn,
+				PrevPackets:        0,
+				PrevBytes:          0,
+				PrevReversePackets: 0,
+				PrevReverseBytes:   0,
+				IsIPv6:             isIPv6,
+			}
+		} else {
+			record.Conn = &conn
+		}
+		fr.recordsMap[key] = record
+		return nil
+	}
+
 	// fr.addOrUpdateFlowRecord method does not return any error, hence no error handling required.
```

Comment:
Change the comment that still says the method name of flow record rather than function name.

Reply:
Indeed. `fr.addOrUpdateFlowRecord` -> `addOrUpdateFlowRecord`